dhall/src/Dhall/Import.hs

```diff
@@ -827,12 +833,19 @@ loadWith expr₀ = case expr₀ of
         Nothing -> do
             return ()
         Just expectedHash -> do
-            let actualHash =
-                    hashExpression _standardVersion (Dhall.Core.alphaNormalize expr)
+            let matches version =
+                    let actualHash =
+                            hashExpression version (Dhall.Core.alphaNormalize expr)
+
+                    in  expectedHash == actualHash
 
-            if expectedHash == actualHash
+            if any matches [ NoVersion, V_5_0_0 ]
```

Comment:
```suggestion
            if any matches [ NoVersion, V_5_0_0, V_4_0_0 ]
```

Spago is on `1.19.1` :)

Reply:
And dhall-to-etlas!

Reply:
Yeah, I meant to include all versions back to 1.0.0, but I just forgot to go back and add them all here 🙂